package cl.forge.programatufuturo.reservaprueba.exceptions;

public class EncriptacionException extends Exception{
    public EncriptacionException(String errorMessage, Throwable e){
        super(errorMessage,e);
    }

}
